pkg/sql/plan2/function/builtin/multi/lpad.go

```diff
@@ -28,6 +30,23 @@ const (
 	ParameterPadString    = int(2)
 )
 
+//for func lpad func, the second para length,it could be type.Bytes or int
```

Comment:
This does not feel right.   Please contact Peng Zhen or MingSong.    I believe you are doing the wrong way, it should go through their type conversion framework.